protostar/commands/test/environments/standard_test_execution_environment.py

```diff
@@ -0,0 +1,14 @@
+from protostar.commands.test.environments.test_execution_environment import (
+    TestExecutionEnvironment,
+    TestExecutionResult,
+)
+
+
+class StandardTestExecutionEnvironment(TestExecutionEnvironment):
```

Comment:
I think people might want to profile fuzz tests and gather averages/boundaries of runs..

In such scenario I think it would be more scalable implementation where `TestExecutionEnvironment` stays with the same role it had and accepts the `profiling` flag, while `FuzzTestExecutionEnvironment` raises an error if this flag is set to `True`.

wdyt?

Reply:
Done